Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+arteriauser    = ENV['ST2USER'] ? ENV['ST2USER']: 'arteriaadmin'
+arteriapasswd  = ENV['ST2PASSWORD'] ? ENV['ST2PASSWORD'] : 'arteriarulz'
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.define "arteria" do |arteria|
+    # Box details
+    arteria.vm.box = "bento/ubuntu-14.04"
+    arteria.vm.hostname = "arteria-dev"
+
+    # Box Specifications
+    arteria.vm.provider :virtualbox do |vb|
+      vb.memory = 1028
+      vb.cpus = 2
+    end
+
+    # NFS-synced directory for pack development
+    # Change "/path/to/directory/on/host" to point to existing directory on your laptop/host and uncomment:
+    config.vm.synced_folder "~/workspace/arteria/arteria-packs", "/arteria-packs", :nfs => true, :mount_options => ['nfsvers=3']
```

Comment:
I was about to say that you should init this path somehow more conveniently, or update the comment and add some instructions about this path in the README. The init could either be user specified, or Vagrant could find the current working directory and just mount that. 

But - that is actually already happening. Vagrant always mounts the directory where `Vagrantfile` resides, under `/vagrant/` on the guest. Can't you use that instead of mounting under `/arteria-packs`? I notice that you're explicitly specifying NFS v3 -- is that a requirement? I remember you briefly mentioning something about NFS earlier and automatic tests...  


Either this path should be initialized by trying to find e.g. the current working directory (which